report/2.Analysis/EmbeddedPlatforms.tex

```diff
@@ -0,0 +1,78 @@
+% !TeX root = ../main.tex
+% This section is meant to introduce single-board computers that are often used for introducing programmers to beginners.
+% The section itself might not be used a whole lot during the design and implementation, as these are not the arduino platform.
+% Still, it is good to give examples of implementations.
+\section{Embedded platform}
+In the following section will examine the embedded platforms, that will be used in the project.
+For the 5th semester of the Software education, a Lego Mindstorms kit is supplied to each project group.
+Therefore the analysis will primarily be based around the Lego Mindstorms platform, with a few additional platforms also considered.
+
+To determine the best platform, the requirements for the project will need to be considered.
+\begin{enumerate}
+	\item At the very least, the platform must be able to perform a targeting calculation, based on a trained machine learning model.
+	\item Be able to interface with motors, most likely Lego.
+	\item Potentially be able to train a machine learning model, which is used for the prediction on which the targeting is based.
+\end{enumerate}
+
+As image recognition is an expensive computation, the current idea is to offload this to an external server or computer, which will compute the current object position and send that to the embedded device.
+
+The current idea is to use the Lego Mindstorms platform for the robotics part of the solution, and use a single-board computer for image object tracking and the machine learning model training.
+
+\subsection{Lego Mindstorms}
+Lego Mindstorms is a programmable computer kit, made by Lego.
+The platform is based around a central control computer, called the Intelligent Brick.
+It also includes a variety of sensors, motors and connection cables.
+Each generation further improves on the concept, adding more computational power and better sensors.
```

Comment:
"improves on the concept" virker abstrakt for mig. Jeg ville nok ændre det til noget ala "Each generation continously removes hardware limitations, by adding..."